index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+  <meta charset='UTF-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+  <title>Tribute Page</title>
+  <link rel='stylesheet' type='text/css' href='css/style.css'>
+	<link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>
+	<link href='https://fonts.googleapis.com/icon?family=Material+Icons' rel='stylesheet'>
+	<script src='https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js'></script>
+	<script src='js/script.js' type='text/javascript'></script>
+</head>
+<body>
+  <main id = 'main'>
+    <h1 id = 'title'>
+      Prof. Wangari Muta Maathai
+    </h1>
+    <blockquote cite = 'https://www.brainyquote.com/quotes/wangari_maathai_396414'>
+      <p>
+        'In a few decades, the relationship between the environment, resources and
+        conflict may seem almost as obvious as the connection we see today between
+        human rights, democracy and peace.'
+      </p>
+      <cite>- Prof. Wangari Muta Maathai</cite>
+    </blockquote>
+    <figure id='img-div'>
+      <img
+        id='image'
+        src='https://www.thoughtco.com/thmb/rDwnnZ0S34PTdzQ26QLA4bDhH8M=/3777x2501/filters:fill(auto,1)/kenyan-activist-wangari-maathai-525194310-58c341ef5f9b58af5c638e4c.jpg'
+        alt='Prof. Wangari Muta Maathai seen standing infront of a rural farm'
+        height = 740px;
+        width = 100%;
+      />
+      <figcaption id='img-caption'>
+        Prof. Wangari Muta Maathai seen standing infront of a rural farm.
+      </figcaption>
+    </figure>
+    <section id = 'tribute-info'>
+      <h3 id = 'headline'>
+        Here's a timeline of Prof. Wangari Muta Maathai's lift:
+      </h3>
+      <ul>
+        <li>
+          <strong>
+            1940
+          </strong>
+          - Born in Ihithe village, Tetu division, Nyeri County, Kenya (then known as Nyeri, Kenya Colony)
+        </li>
+        <li>
+          <strong>
+            1943
+          </strong>
+          - Family relocates to white-owned farm in the Rift Valley, near Nakuru town
+        </li>
+        <li>
+          <strong>
+            1947
+          </strong>
+          - Returned to Ihithe with her mother and two brothers to attend Ihithe Primary School
+        </li>
+        <li>
+          <strong>
+            1951
+          </strong>
+          - Moved o St. Cecilia's Intermediate Primary School & joined <a target = '_blank' href = 'https://en.wikipedia.org/wiki/Legion_of_Mary'>the Legion of Mary</a>
```

Comment:
For all a tags, please include "rel=noopener", disown-opener checks if the rel attribute is specified with both the noopener and noreferrer values (or only noopener if all the [targeted browsers](https://webhint.io/docs/user-guide/configuring-webhint/browser-context/) support it) on a and area elements that have target="_blank" and link to other origins. You can read more about it [here](https://webhint.io/docs/user-guide/hints/hint-disown-opener/)